packages/lib-classifier/src/components/Classifier/components/Feedback/FeedbackModal.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { inject, observer } from 'mobx-react'
+import { Button, Box } from 'grommet'
+import { Modal } from '@zooniverse/react-components'
+import counterpart from 'counterpart'
+import en from './locales/en'
+
+import SubjectViewer from '../SubjectViewer'
+
+counterpart.registerTranslations('en', en)
+
+function storeMapper(stores) {
+  const { 
+    hideFeedback,
+    messages,
+    showModal
+  } = stores.classifierStore.feedback
+  return {
+    hideFeedback,
+    messages,
+    showModal
+  }
+}
+
+@inject(storeMapper)
+@observer
+class FeedbackModal extends React.Component {
+  render () {
+    const label = counterpart('FeedbackModal.label')
+    const { hideFeedback, messages, showModal } = this.props
+
+    if (showModal) {
+      return (
+        <Modal
+          active={showModal}
+          closeFn={() => hideFeedback()}
+          title={label}
+        >
+          <>
+            <Box
+              height="medium"
+              overflow="auto"
+            >
+              <SubjectViewer />
```

Comment:
Do we have a subject viewer component that we can use to display feedback? This seems like a major piece of work.